manifests/m/Maltego/Maltego/4.5.0/Maltego.Maltego.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Maltego.Maltego
+PackageVersion: 4.5.0
+InstallerType: nullsoft
+Dependencies:
+  ExternalDependencies:
+  - Oracle.JDK.21
+AppsAndFeaturesEntries:
+- DisplayVersion: "4"
+  ProductCode: Maltego
+Installers:
+- Architecture: x86
+  InstallerUrl: https://downloads.maltego.com/maltego-v4/windows/MaltegoSetup.v4.5.0.exe
+  InstallerSha256: 58B90210E620C0FEFBB8D4C729D9C7DD2720A5600CCD44D82B3E01BB3C3D439C

```

Beside this installer file, Maltego provides a "64-bits JRE bundled" installer. Shall I remove the `ExternalDependencies` and use the "JRE bunled" version instead?

```suggestion
  InstallerUrl: https://downloads.maltego.com/maltego-v4/windows/MaltegoSetup.JRE64.v4.5.0.exe
  InstallerSha256: 10C9DEA7093BB6C2D9B29C72FC2DCAEE10937802C547F1FBBDAEBF33EFB5AD94
```